### ISSUE TITLE
Build Failure doesnt expand output

### ISSUE THREAD

**Issue by plessbd** [Shippable/support#4590]
https://app.shippable.com/github/ubccr/xdmod/runs/2658/1/console

The build is not expanding the output like it usually does, not sure if this is something with how we are doing the tests or if it is something else

However, I do notice that these issues are specifically also outputting:

```
.__SH__CMD__START__|{"type":"cmd","sequenceNumber":"1538570534","id":"69ee81a1-5fbc-4832-a42b-2100d1490739"}|cp ./configuration/portal_settings.ini ./configuration/portal_settings.ini.old

  ./open_xdmod/modules/xdmod/integration_tests/runtests.sh --junit-output-dir `pwd`/shippable/testresults/

.__SH__CMD__START__|{"type":"cmd","sequenceNumber":"1538570534","id":"85250267-597b-4319-9c4f-c7c953a22cda"}|./open_xdmod/modules/xdmod/integration_tests/runtests.sh --junit-output-dir `pwd`/shippable/testresults/
```
which looks like it is used for the code folding...

**Comment by a-murphy** [comment]
Yes, the `__SH__CMD__START__` line is used to determine where the output of a command starts.  Since that information ended up on the same line as a `.`, it wasn't recognized as that marker and appeared in the output instead.  We'll look into what happened to allow the start marker to print on the same line as console output in these logs and let you know what we find.

**Comment by plessbd** [comment]
Ahh didnt notice that.
That might be something to do with how we are running the phpunit tests, I will see if I can make sure things get output properly there, thank you.
